sdk/go/common/resource/plugin/provider_plugin.go

```diff
@@ -104,15 +102,31 @@ func NewProvider(host Host, ctx *Context, pkg tokens.Package, version *semver.Ve
 			}
 		}
 	}
+	port, err := strconv.Atoi(optAttach)
```

Comment:
Shouldn't this check that optAttach != ""

Reply:
Good catch.